tests/actors.rs

```diff
@@ -0,0 +1,125 @@
+use crate::common::spawn_app;
+use sqlx::{Pool, Postgres};
+use std::collections::HashMap;
+
+mod common;
+
+// We use sqlx::test so sqlx will create a database instance for the test,
+// set it up based on the migration data, and shut it down upon success.
+// The pool passed to the function is that created database.
+#[sqlx::test]
+async fn get_actors_works(pool: Pool<Postgres>) {
```

Comment:
nit: "works" is kind of the point of every test. I'd suggest either leaving it off or being more specific (one possible option being something like `get_actors_when_one_exists`).  
That also clarifies what it is and isn't testing. For instance, does this endpoint still work with no actors in the table? Does it work with more than one actor?  
  
That example follows a `<function name> when <precondition>` pattern.  
Another more verbose way I've seen used is `<function name> when <precondition> expect <outcome>` that gives names like this `get_actors_when_no_actors_expect_empty_response`